app-drawer-layout/app-drawer-layout.html

```diff
@@ -138,6 +138,14 @@
           value: '640px'
         },
 
+        /**
+         * True if the app-drawer-layout should fit the size of its container.
+         */
+        fullbleed: {
```

Comment:
I'm assuming this is needed only for styling - is there a performance hit for making this a property? (Though this is just for `app-drawer-layout`, so this may matter less.)

cc @frankiefu